explainer.md

```diff
@@ -42,10 +45,12 @@ The proposed API is very similar to the Fullscreen API as they have similar prop
 ```
 partial interface HTMLVideoElement {
   Promise<PictureInPictureWindow> requestPictureInPicture();
+  Promise<void> setPictureInPictureControls(FrozenArray<PictureInPictureControl> pipControls);
 
   // On the fullscreen API, they live on the Document.
   attribute EventHandler onenterpictureinpicture;
   attribute EventHandler onleavepictureinpicture;
+  attribute PictureInPictureEventHandler onpictureinpicturecontrolsclick;
```

Comment:
Nit: Should this be `PictureInPictureControlsEventHandler` to make it clear it won't be the one for `onenterpictureinpicture` and `onleavepictureinpicture`?

Reply:
Updated.